app/assets/stylesheets/common/base/menu-panel.scss

```diff
@@ -90,6 +90,72 @@
   }
 }
 
+.user-menu.revamped {
+  right: 0;
+  width: 320px;
+  padding: 0;
+
+  .panel-body-bottom {
+    flex: 0;
+  }
+
+  .menu-tabs-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    border-left: 1px solid var(--primary-low);
+  }
+
+  .tabs-list {
+    display: flex;
+    flex-direction: column;
+
+    .btn {
+      display: flex;
+      padding: 0.857em;
```

Comment:
Is it possible to use use a more "standard" value ?

Reply:
Haha arriving at this value was an adventure 😅 This padding controls the width of the menu tabs which need to match the width of the avatar so we really can't deviate from it:

![image](https://user-images.githubusercontent.com/17474474/179152880-2a6a5f8b-d0e5-45ba-842e-42cc7f15e2c4.png)

Reply:
Apologies, I'm really not sure how to come up with non-magic value that aligns with the avatar. The avatar in the header has had magic width and padding for 4-5 years, see:

https://github.com/discourse/discourse/blob/0760b249ffe39a9ff57b7e36b6304e63a5082d7e/app/assets/stylesheets/common/base/header.scss#L119-L121

Are you suggesting I change the avatar's width and padding values to something less magic? Or is your objection to having magic values for padding, but it's ok for width/height?

Reply:
If the menu has a fixed width, you don't need to set a padding on controls and just rely on the max width set by the avatar no ?

Reply:
Ok dont lose more time on this :) thanks for checking, will look when it's live to see if I think of something different